arrange_schedule.py

```diff
@@ -1347,28 +1348,12 @@ def set_system_log(json_obj):
     return return_msg
         
 def expire_data_check(raw_time):
-    shutdown = 0
-    receive_obj = {}
-    try:
-        newpid = os.fork()
-        if newpid == 0: #child
-            shutdown = 1
-            receive_obj = expire_data_check_()
-            if receive_obj["result"] == "success":
-                "DO NOTHING"
-            else :
-                receive_obj["error"] = "expire_data_check : " + receive_obj["error"]
-                set_system_log(receive_obj)
-            os._exit(0)
-        else: #Parent
-            alarm_expire_data_check = raw_time + 1800.0
-    except:
-        receive_obj["result"] = "fail"
-        receive_obj["error"] = "fork1 error"
+    receive_obj = expire_data_check_()
+    if receive_obj["result"] == "success":
+        "DO NOTHING"
+    else :
+        receive_obj["error"] = "expire_data_check : " + receive_obj["error"]
```

Comment:
If all the string representation should using `.format()` method, maybe line 1355 could change to : 
`receive_obj["error"] = "expire_data_check : {message}".format(message=receive_obj["error"])`